src/components/Connection/Desktop/CreateAccount.vue

```diff
@@ -0,0 +1,497 @@
+
+  get iconPasswordStyle(): string {
+    return this.hiddenInput ? 'basic-eye-no-24' : 'basic-filterlist-24';
+  }
+
+  get btnTypeConfirmStep(): string {
+    if (this.seedPhraseToCompare.length === this.PHRASE_LENGTH) {
+      return 'primary';
+    }
+    return 'secondary';
+  }
+
+  get isInputsNotFilled(): boolean {
+    return !this.accountName || !this.accountPassword || !this.accountPasswordConfirm;
+  }
+
+  getKey(word, idx): string {
+    const cssClass = 'word' + Number(idx + 1);
+    this.seedPhraseBoundToClass.set(word, cssClass);
+    return cssClass;
+  }
+
+  get seedPhrase(): string {
+    const { seed } = api.createSeed();
+    return seed;
+  }
+
+  get seedPhraseAsArray(): Array<string> {
+    const seedArray = this.seedPhrase.split(' ').map((word) => word.toUpperCase());
+    return seedArray;
+  }
+
+  get randomizedSeedPhrase(): Array<string> {
+    return [...this.seedPhraseAsArray].sort(() => Math.random() - 0.5);
+  }
+
+  toggleVisibility(): void {
+    this.hiddenInput = !this.hiddenInput;
+  }
+
+  chooseWord(e: Event): void {
+    const clickedWord = e.currentTarget;
+    const targetClass = (clickedWord as HTMLElement).classList[1];
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    this.$refs[`${targetClass}`]![0].classList.add('word--hidden')!;
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const clickedWordText = (clickedWord as HTMLElement).textContent!.trim();
+    this.$nextTick(() => {
+      this.seedPhraseToCompare.push(clickedWordText);
+    });
+  }
+
+  discardWord(e: Event): void {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const clickedWordText = (e.currentTarget as HTMLElement).textContent!.trim();
+    const wordToDiscard = this.seedPhraseBoundToClass.get(clickedWordText);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    this.$refs[`${wordToDiscard}`]![0].classList.remove('word--hidden');
+    this.seedPhraseToCompare = this.seedPhraseToCompare.filter((word) => word !== clickedWordText);
+  }
+
+  discardAllWords(): void {
+    const wordsHtmlDOM = Array.from(new Array(this.PHRASE_LENGTH), (_, idx) => `word${idx + 1}`);
+    wordsHtmlDOM.map((value) => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      this.$refs[value]![0].classList.remove('word--hidden');
+    });
+    wordsHtmlDOM.map((value) => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      this.$refs[value]![0].classList.add('login__random-word--incorrect');
+    });
+    setTimeout(() => {
+      wordsHtmlDOM.map((value) => {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        this.$refs[value]![0].classList.remove('login__random-word--incorrect');
+      });
+    }, 2000);
+  }
+
+  async handleCopy(): Promise<void> {
+    await copyToClipboard(this.seedPhraseAsArray.map((w) => w.toLowerCase()).join(' '));
```

Comment:
don't we have ready for copy seedPhrase?

Reply:
fixed